src/views/product-landing/schema.ts

```diff
@@ -187,6 +162,60 @@ const ProductLandingGetStartedSchema = Joi.alternatives().try(
 	})
 )
 
+interface CalloutContentBlock extends CalloutCardProps {
+	type: 'callout'
+}
+
+/**
+ * Roll up of all block types
+ */
+export type ProductLandingContentBlock =
+	| HeadingContentBlock
+	| TutorialCardsContentBlock
+	| CollectionCardsContentBlock
+	| LinkedCardsContentBlock
+	| CalloutContentBlock
+const ProductLandingContentBlockSchema = Joi.object({
+	type: Joi.string()
+		.required()
+		.valid('heading', 'tutorial_cards', 'linked_cards'),
+})
+	.when(Joi.object({ type: 'callout' }).unknown(), {
+		then: Joi.object({
```

Comment:
⛏️ Nit: if we want, we could declare another schema for this, and keep things grouped by block type (types & schemas for each block type together, with the broader schema assembling a bunch of smaller ones rather than declaring things inline).

Reply:
Good call, will update